inc/config.h

```diff
@@ -46,18 +46,35 @@
 // ############################### SERIAL DEBUG ###############################
 
 //#define DEBUG_SERIAL_USART3         // right sensor board cable, disable if I2C (nunchuck or lcd) is used!
+#define DEBUG_SERIAL_USART2         // left sensor board cable, disable if ADC or PPM control is used!
 //#define DEBUG_SERIAL_SENSOR         // send to USART3 sensor board, without framing, at the CONTROL_SENSOR_BAUD rate
-#define DEBUG_BAUD       115200     // UART baud rate
 //#define DEBUG_SERIAL_SERVOTERM
+#define DEBUG_BAUD       19200      // UART baud rate
 #define DEBUG_SERIAL_ASCII          // "1:345 2:1337 3:0 4:0 5:0 6:0 7:0 8:0\r\n"
+#define UART_SEND_METHOD_2          // Change beetwen 2 methods of sending data
```

Comment:
I think method 2 is best but I've implemented the 2 methods if one is not working.
See [here](#diff-8cb700748da1954a230281220aa9e5daR102) and [here](#diff-4964764a2e677a55c2f4207a995c3ce7R145)